src/Uno.UI/UI/Xaml/Media/ImageSource.macOS.cs

```diff
@@ -86,6 +86,23 @@ private static NSImage OpenBundleFromString(string bundle)
 			return null;
 		}
 
+		private static NSImage OpenResourceFromString(string name)
+		{
+			if (name.HasValueTrimmed())
+			{
+				var extension = Path.GetExtension(name);
+				var fileName = name.Replace(extension, string.Empty);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'name.Replace' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5280119)